client/jetpack-cloud/sections/partner-portal/issue-multiple-licenses-form/index.tsx

```diff
@@ -75,7 +75,10 @@ export default function IssueMultipleLicensesForm( {
 		if ( selectedProductSlugs.find( ( product ) => isJetpackBundle( product ) ) ) {
 			issueLicenses();
 		}
-	}, [ selectedProductSlugs, issueLicenses ] );
+		// Do not update the dependency array with issueLicenses since
+		// it gets changed on every product change, which triggers this `useEffect` to run infinitely.
+		// eslint-disable-next-line react-hooks/exhaustive-deps
+	}, [ selectedProductSlugs ] );
```

Comment:
I wonder, is it better to have this entire logic happen inside `onSelectProduct()`? It could be done on line 59, instead of updating `selectedProductSlugs` it could just call `issueLicenses()`.

Reply:
Currently, the arguments are being passed to the hook directly. We could refactor the hook to accept the arguments in the function which handles issuing licenses. We can address this in a different PR; I have created a [ticket](1202619025189113-as-1203377005312989) for this.